<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parents</class>
 <widget class="QMainWindow" name="parents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>501</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>702</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parent</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ssl.qrc">
    <normaloff>:/P.png</normaloff>:/P.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>684</width>
        <height>462</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>800</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="p_grades">
       <attribute name="title">
        <string>Grades</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>481</width>
          <height>271</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
       <widget class="QComboBox" name="gradeType_comboBox_6">
        <property name="geometry">
         <rect>
          <x>231</x>
          <y>60</y>
          <width>121</width>
          <height>24</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Choose</string>
        </property>
        <item>
         <property name="text">
          <string>Test</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Assignment</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="getGrades_pushButton_4">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>60</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Get</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="p_cProfile">
       <attribute name="title">
        <string>Childs Profile</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>361</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="childName_lineEdit_8">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>201</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Student Id:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="childID_lineEdit_7">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>90</y>
          <width>91</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Age:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="childAge_lineEdit_6">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="childPP_label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>img</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="p_tProfile">
       <attribute name="title">
        <string>Teacher Profile</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_5">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>150</y>
          <width>181</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Id:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="teacherid_lineEdit_5">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="teacherPP_label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>161</width>
          <height>171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_6">
        <property name="geometry">
         <rect>
          <x>204</x>
          <y>70</y>
          <width>361</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="teacherName_lineEdit_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>301</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_12">
         <item row="0" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Phone No:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="teacherPhone_lineEdit_12">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_8">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>110</y>
          <width>331</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Subject:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="teacherSubject_lineEdit_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QComboBox" name="selectTeacher_comboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>371</width>
          <height>24</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Select Teacher</string>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>400</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../Downloads/logout.png</normaloff>../../../../Downloads/logout.png</iconset>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="t_messages">
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <widget class="Line" name="line_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>661</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>New Message</string>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>621</width>
          <height>421</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>619</width>
           <height>419</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>702</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ssl.qrc"/>
 </resources>
 <connections/>
</ui>
